package.json

```diff
@@ -1,13 +1,16 @@
 {
   "name": "@oat-sa/tao-extension-release",
-  "version": "0.4.1",
+  "version": "0.5.0",
```

Comment:
Isn't it a breaking change ?
the public API (the command line) has been modified (from `taoRelease` to `taoRelease action`)

Reply:
You are right, we should go to 1.0.0